apps/web/src/contexts/globalLayout/GlobalSidebar/SidebarDrawerContext.tsx

```diff
@@ -0,0 +1,83 @@
+import {
```

Comment:
Any reason to keep this separate from `GlobalLayoutContext`? Seems like they're related enough to just shove the state in there

Reply:
Fair enough. Let's keep it this way for now. Can revisit later.